packages/host/app/components/operator-mode/overlays.gts

```diff
@@ -113,23 +105,52 @@ export default class OperatorModeOverlays extends Component<Signature> {
     </style>
   </template>
 
+  @tracked isHoverOnCard = new TrackedWeakMap<RenderedLinksToCard, boolean>();
+  isEventsRegistered = new TrackedWeakMap<RenderedLinksToCard, boolean>();
```

Comment:
Perhaps `areHoverEventsRegistered` could be a more descriptive variable for this map?